### ISSUE TITLE
Some questions about the code

### ISSUE THREAD

**Issue by Zawo1995** [MIRALab-USTC/KGE-HAKE#1]

**Comment by Zawo1995** [comment]
Thanks for sharing the  code HAKE，I have some question about about the code : 
the  loss  computing in the code write is 
` negative_score = (F.softmax(negative_score * args.adversarial_temperature, dim=1).detach()* F.logsigmoid(-negative_score)).sum(dim=1)` 
but in the paper,the formula is use the f_r to compute softmax,not the '(gamma - d_r)' which computed from the model 

![111](https://user-images.githubusercontent.com/48239524/70907446-d8c66600-2043-11ea-9df3-7b36c661faac.png)
Is that the gamma makes no influence to softmax function?



Sorry for asking so shallow question :)

Looking forward to your response :)

**Comment by zhanqiuzhang** [comment]
As $f_r=-d_r$, we have that
![image](https://user-images.githubusercontent.com/22275172/70913295-b9820580-2050-11ea-9aef-75ba233056f2.png)
That is, the gamma makes no influence on the softmax function.

**Comment by Zawo1995** [comment]
Thanks a lot
